code/game/objects/effects/decals/Cleanable/tracks.dm

```diff
@@ -118,13 +121,15 @@ GLOBAL_LIST_EMPTY(fluidtrack_cache)
 		FP.blood_state = H.blood_state
 		FP.bloodiness = H.bloody_feet[H.blood_state] - BLOOD_LOSS_IN_SPREAD
 		FP.basecolor = H.feet_blood_color
+		FP.base_alpha = H.bloody_feet["blood_alpha"]
 		if(H.blood_DNA)
 			FP.blood_DNA = H.blood_DNA.Copy()
 	else if(istype(A, /obj/item/clothing/shoes))
 		var/obj/item/clothing/shoes/S = A
 		FP.blood_state = S.blood_state
 		FP.bloodiness = S.bloody_shoes[S.blood_state] - BLOOD_LOSS_IN_SPREAD
 		FP.basecolor = S.blood_color
+		FP.base_alpha = S.bloody_shoes["blood_alpha"]
```

Comment:
```suggestion
		FP.base_alpha = S.bloody_shoes[BLOOD_STATE_BASE_ALPHA]
```
use your defines. do this for all instances of this please.

Reply:
haha ok =